submissions/krylenger/memory-pair-game/app.js

```diff
@@ -0,0 +1,244 @@
+const orelSong = new Audio('music/orel.mp3');
+const gopgopSong = new Audio('music/gopgop.mp3');
+const tanciSong = new Audio('music/tanci.mp3');
+const cadilacSong = new Audio('music/cadilac.mp3');
+const dimSong = new Audio('music/dim.mp3');
+
+let playedSongs = [];
+
+const cards = [
+    {
+        key: 1,
+        text: 'orel',
+        musicCard: false
+    },
+    {
+        key: 1,
+        text: '$',
+        musicCard: true,
+        song: orelSong
+    },
+    {
+        key: 2,
+        text: 'gopgop',
+        musicCard: false
+    },
+    {
+        key: 2,
+        text: '$',
+        musicCard: true,
+        song: gopgopSong
+    },
+    {
+        key: 3,
+        text: 'tanci',
+        musicCard: false
+    },
+    {
+        key: 3,
+        text: '$',
+        musicCard: true,
+        song: tanciSong
+    },
+    {
+        key: 4,
+        text: 'cadilac',
+        musicCard: false
+    },
+    {
+        key: 4,
+        text: '$',
+        musicCard: true,
+        song: cadilacSong
+    },
+    {
+        key: 5,
+        text: 'dim',
+        musicCard: false
+    },
+    {
+        key: 5,
+        text: '$',
+        musicCard: true,
+        song: dimSong
+    }
+]
+
+const main = document.querySelector('.main');
+let mainGridContainer = document.querySelector('.main__grid-container');
```

Comment:
you can also make this variable as const as well. 
Your `mainGridContainer` variable is a DOM node. It's like a simple object. You can define it as the const and change any of the properties of it.

Reply:
✅ Done

Reply:
got it. ok